sympy/physics/continuum_mechanics/beam.py

```diff
@@ -290,6 +290,12 @@ def join(self, beam, via="fixed"):
             new_beam._hinge_position = self.length
             return new_beam
 
+    def polar_moment(self,axis=None):
+        if axis==None or axis=='x':
+            if isinstance(self._second_moment,int) or isinstance(self._second_moment,Symbol):
+                return 2*self.second_moment
+            return self._second_moment[0]+self._second_moment[1]
+
```

Comment:
Please provide a docstring with examples. I'm not sure why you need to pass in an axis.

Reply:
Well generally we find polar moment of inertia about the axis along the length of the beam(about the x-axis here). But we can also find it about the other two axes if we know the shape of beam and dimensions. The function  I have written is for x-axis only right now.
Should I delete the axis part?

Reply:
Yes, if it doesn't do anything.

Reply:
Done. Can it be merged?